# import-replace/import-replace

ðŸ”§ This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).

<!-- end auto-generated rule header -->

## Rule Details

[
  {
    form: 'lodash',
    to: 'lodash-es'
  }
]

Examples of **incorrect** code for this rule:

```js

import _ from 'lodash';

```

Examples of **correct** code for this rule:

```js

import _ from 'lodash-es';

```
